pkg/server/metrics.go

```diff
@@ -82,6 +85,21 @@ var _initMetricsFunc = sync.OnceValue[*metrics](func() *metrics {
 		Help: "API Request size on HTTP calls",
 	}, []string{"code", "method"})
 
+	m.grpcLatency = f.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
Are we not already getting a lot of these from the default server metrics?

Reply:
I think we can enable this? https://github.com/grpc-ecosystem/go-grpc-middleware/blob/0e172e476feda43f0e83cc894fe2d636c1d820fa/providers/prometheus/server_options.go#L37